metadrive/constants.py

```diff
@@ -189,7 +189,7 @@ def collision_rules(cls):
             (cls.Sidewalk, cls.Sidewalk, False),
             (cls.Sidewalk, cls.LidarBroadDetector, False),
             (cls.Sidewalk, cls.TrafficObject, True),
-            (cls.Sidewalk, cls.TrafficParticipants, True),
+            (cls.Sidewalk, cls.TrafficParticipants, False),  # don't allow sidewalk contact
```

Comment:
Why do that? I think this decision will affect a lot of things. Should remove this to a new PR.

Reply:
No. It only affects the collisions between traffic participants (pedestrians and cyclist) and sidewalk collision. Disabling it to avoid object falling down